pkg/operator/ceph/object/objectstore.go

```diff
@@ -65,6 +68,12 @@ func createObjectStore(context *Context, metadataSpec, dataSpec model.Pool, serv
 	if err != nil {
 		return errors.Wrapf(err, "failed to create object store realm")
 	}
+
+	err = enableRGWDashboard(context)
```

Comment:
Note that `createObjectStore()` could be called multiple times for the same object store. For example, each time the operator restarts it will run a full orchestration. Or is the object CR is updated, this will be called again to reconcile the object store settings. Are the commands in `enableRGWDashboard()` idempotent? In other words, if it's already configured once will it be ok to run this same method again?

Reply:
Added a check whether get-rgw-api-access-key before setting the new one